doc/sphinx/source/ops/dequantize.rst

```diff
@@ -0,0 +1,66 @@
+.. dequantize.rst: 
+
+##########
+Dequantize
+##########
+
+.. code-block:: cpp
+
+   Dequantize // Maps quantized input to real output using scale and offset
+
+Description
+===========
+
+Produces a tensor of element type ``type`` and the same shape as ``input``
+where the value of each coordinate (i) of ``output`` is the corresponding coordinate of 
+``input`` plus ``offset`` quantity multiplied by ``scale``.  
+The coordinate (j) of ``scale`` and ``offset`` is the coordinate of ``output``
+projected along ``axes``.
+
+Inputs
+------
+
++-----------------+-------------------------+---------------------------------------+
+| Name            | Element Type            | Shape                                 |
++=================+=========================+=======================================+
+| ``input``       | is_quantized()          | Any                                   |
```

Comment:
Not sure if this is adhering to an existing convention, but if not I'd prefer something like "Any quantized type" instead of `is_quantized()`.